great_expectations_provider/example_dags/bigquery_example.py

```diff
@@ -0,0 +1,61 @@
+"""
+A DAG that demonstrates implementation of the GreatExpectationsBigQueryOperator. 
+
+Note: you wil need to reference the necessary data assets and expectations suites in your project. You can find samples available in the provider source directory.
+
+Steps to run:
```

Comment:
See my comment re: defaulting to Astro. Also, for maintainability, I would prefer to say "see the steps in the README.md" and not repeat them here, and only add the bigquery connection at the end.